ios/PhotoLibrary/RNPermissionHandlerPhotoLibrary.m

```diff
@@ -23,14 +31,30 @@ - (void)checkWithResolver:(void (^ _Nonnull)(RNPermissionStatus))resolve
       return resolve(RNPermissionStatusDenied);
     case PHAuthorizationStatusAuthorized:
       return resolve(RNPermissionStatusAuthorized);
+    case PHAuthorizationStatusLimited:
```

Comment:
`PHAuthorizationStatusLimited` is not available on iOS < 14, no?

Reply:
I've tested it on simulator and works on both iOS 14 and iOS 13 without problem!

Reply:
The simulator is horrible on iOS, have you tested it on a real device?

Reply:
https://developer.apple.com/documentation/photokit/phauthorizationstatus/phauthorizationstatuslimited?language=occ Support is actually iOS 14+